salt/states/ssh_known_hosts.py

```diff
@@ -151,6 +151,11 @@ def present(
             ret["comment"] = f"ssh.check_known_host error: {err}"
             return ret
 
+        if isinstance(result, dict) and "error" in result:
+            ret["result"] = False
+            if "error" in result:
```

Comment:
error is in result; we already checked that in the condition on line 154 `if isinstance(result, dict) and "error" in result`, so remove this redundant condition.

Reply:
Nice, thanks.

Reply:
Thanks for catching it!